dev/integration_tests/flutter_gallery/lib/gallery/about.dart

```diff
@@ -25,7 +25,7 @@ class _LinkTextSpan extends TextSpan {
   _LinkTextSpan({ super.style, String? url, String? text }) : super(
     text: text ?? url,
     recognizer: TapGestureRecognizer()..onTap = () {
-      launch(url!, forceSafariVC: false);
+      launchUrlString(url!);
```

Comment:
@stuartmorgan I updated a few deprecated APIs in this directory, could you please have a look - I am not sure if parameters match correctly.